deployment/src/main/java/io/quarkiverse/univocityparsers/deployment/UnivocityParsersProcessor.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkiverse.univocityparsers.deployment;
+
+import org.jboss.jandex.DotName;
+import org.jboss.logging.Logger;
+
+import com.univocity.parsers.conversions.Conversion;
+
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.IndexDependencyBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
+
+class UnivocityParsersProcessor {
+
+    private static final Logger log = Logger.getLogger("io.quarkiverse.univocityparsers");
+
+    private static final String FEATURE = "univocity-parsers";
+    private static final DotName CONVERSION_INTERFACE = DotName.createSimple(Conversion.class.getName());
+
+    @BuildStep
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(FEATURE);
+    }
+
+    @BuildStep
+    IndexDependencyBuildItem indexUnivocityParserJar() {
+        return new IndexDependencyBuildItem("com.univocity", FEATURE);
```

Comment:
It's a coincidence that the feature name and the artifact ID are the same, maybe refrain from reusing the same constant for clarification purposes

Reply:
Agreed. It's a bad idea.

Reply:
a take example on other quarkiverse projects :-) my newbie in the group